include/gfxbase.h

```diff
@@ -65,6 +65,7 @@
 #pragma once
 
 #include <stdexcept>
+#include <Arduino.h>
```

Comment:
With the 3.9.x FastLED compilation issues having been worked around, is this include still necessary?

Reply:
Removed